controlplane/src/core/env.schema.ts

```diff
@@ -39,4 +39,10 @@ export const envVariables = z.object({
   S3_STORAGE_URL: z.string(),
   SMTP_USERNAME: z.string().optional(),
   SMTP_PASSWORD: z.string().optional(),
+  /**
+   * Billing
+   */
+  STRIPE_SECRET_KEY: z.string().optional(),
+  STRIPE_WEBHOOK_SECRET: z.string().optional(),
+  DEFAULT_PLAN: z.string().optional(), // eg developer@1
```

Comment:
Do we treat the features entirely optionally in the backend and studio?

Reply:
Yep this way it supports on-prem.